server/router/location.ts

```diff
@@ -1,52 +1,57 @@
 import express from 'express';
 import uuid from 'uuid/v1';
-import AWS from 'aws-sdk';
-import config from '../config';
+import dynamoose from 'dynamoose';
 
 const router = express.Router();
 
-AWS.config.update(config);
-const docClient = new AWS.DynamoDB.DocumentClient();
-
-type Location = {
+type LocationType = {
   id: string,
-  location: string,
+  name: string,
+  address: string,
 };
 
+const schema = new dynamoose.Schema({
+  id: String,
+  name: String,
+  address: String,
+});
+
+export const Locations = dynamoose.model('Locations', schema, { create: false });
+
 // Get a location by ID in Locations table
 router.get('/:id', (req, res) => {
-  const { id } = req.params;
-  const params = {
-    TableName: 'Locations',
-    Key: { id },
-  };
-  docClient.get(params, (err, data) => {
-    if (err) {
-      res.send({ err });
-    } else {
-      res.send(data);
-    }
-  });
+  // const { id } = req.params;
+  // const params = {
+  //   TableName: 'Locations',
+  //   Key: { id },
+  // };
+  // docClient.get(params, (err, data) => {
+  //   if (err) {
+  //     res.send({ err });
+  //   } else {
+  //     res.send(data);
+  //   }
+  // });
```

Comment:
If anywhere in this PR, there is something that works currently but has not been replaced by dynamoose, leave it as is. Do not remove functionality by commenting things out. Makes the diff much larger than it needs to be right now. Plus some might be in use by the devs working on the apps for testing purposes.

Reply:
oh thanks for letting me know, i commented this out temporarily but i guess i forgot to put it back